software/paie111_ansible/configure_spectrum_conductor.yml

```diff
@@ -26,20 +26,20 @@
 - name: Activate dli and install dependencies
   shell: "source /opt/anaconda2/bin/activate dli && {{ item }}"
   loop:
-    - "/opt/anaconda2/bin/conda install wheel==0.31.1"
-    - "pip install --index-url http://{{ host_ip.stdout }}/repos/pypi/simple \
-       setuptools==39.0.1 --trusted-host {{ host_ip.stdout }}"
     - "/opt/anaconda2/bin/conda install cython==0.25.2 h5py==2.7.0 \
```

Comment:
I don't think it makes any difference since we're controlling the available packages, but the PAIE 1.1.1 instructions now say to use the `--no-update-dependencies` flag when installing these packages to the dli conda environment.

Reply:
added